src/components/sales/Cart.tsx

```
@@ -178,23 +186,93 @@ export default function Cart({ items, onClear, onUpdateQuantity, onRemove }: Pro
+        {showPaymentOptions && (
+          <div className="mt-6 bg-gray-100 dark:bg-gray-600 p-4 rounded-lg">
+            <h4 className="font-semibold mb-3 text-black dark:text-white">
+              Select Payment Method:
+            </h4>
+            <div className="flex gap-3 mb-4">
+              <button
+                onClick={() => setSelectedMethod('mobile')}
+                className={`px-4 py-2 rounded-lg ${
+                  selectedMethod === 'mobile' ? 'bg-blue-500 text-white' : 'bg-white'
+                }`}
+              >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)